sdk/keyvault/azure-keyvault-keys/migration_guide.md

```diff
@@ -0,0 +1,271 @@
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.keys import KeyClient
+
+credential = DefaultAzureCredential()
+
+key_client = KeyClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+```
+
+You can also create a `CryptographyClient` to enable cryptographic operations (encrypt/decrypt, wrap/unwrap, sign/verify) using a particular key.
+
+```python
+from azure.keyvault.keys.crypto import CryptographyClient
+
+key = key_client.get_key("key-name")
+crypto_client = CryptographyClient(key=key, credential=credential)
+```
+
+### Async operations
+
+The modern `azure-keyvault-keys` library includes a complete async API supported on Python 3.5+. To use it, you must first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/). See [azure-core documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#transport) for more information.
+
+Async operations are available on async clients, which should be closed when they're no longer needed. Each async client is an async context manager and defines an async `close` method. For example:
+
+```python
+from azure.identity.aio import DefaultAzureCredential
+from azure.keyvault.keys.aio import KeyClient
+
+credential = DefaultAzureCredential()
+
+# call close when the client is no longer needed
+client = KeyClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+...
+await client.close()
+
+# alternatively, use the client as an async context manager
+client = KeyClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+async with client:
+  ...
+```
+
+### Create a key
+
+In `azure-keyvault` you could create a key by using `KeyVaultClient`'s `create_key` method, which required a vault endpoint, key name, and key type. This method returned a `KeyBundle` containing the key.
+
+```python
+# create an RSA key
+key_bundle = client.create_key(
+    vault_base_url="https://my-key-vault.vault.azure.net/",
+    key_name="key-name",
+    kty="RSA"
+)
+key = key_bundle.key
+
+# create an elliptic curve key
+key_bundle = client.create_key(
+    vault_base_url="https://my-key-vault.vault.azure.net/",
+    key_name="key-name",
+    kty="EC"
+)
+key = key_bundle.key
+```
+
+Now in `azure-keyvault-keys` there are multiple ways to create keys. You can provide a key name and type to the general `create_key` method, or provide just a name to `create_rsa_key` or `create_ec_key`. These methods all return the created key as a `KeyVaultKey`.
+
+```python
+# create a key with specified type
+key = key_client.create_key(name="key-name", key_type="oct")
```

Comment:
"oct" will get an error from a standard Key Vault. Also may be useful to show the `KeyType` enum here:
```suggestion
from azure.keyvault.keys import KeyType

# create a key with specified type
key = key_client.create_key(name="key-name", key_type=KeyType.ec)
```